plugins/reach/lib/model/enums/VendorServiceFeature.php

```diff
@@ -21,4 +21,5 @@ interface VendorServiceFeature extends BaseEnum
 	const VIDEO_ANALYSIS                  = 14;
 	const MODERATION                      = 15;
 	const METADATA_ENRICHMENT             = 16;
+	const SENTIMENT_ANALYSIS              = 18;
```

Comment:
```suggestion
	const SENTIMENT_ANALYSIS              = 17;
```

Reply:
Another open PR uses 17, which is why I use 18 here.
https://github.com/kaltura/server/pull/13273